core-android/src/main/java/com/uber/sdk/android/core/auth/LoginManager.java

```diff
@@ -178,7 +179,7 @@ public void login(final @NonNull Activity activity) {
         if (ssoDeeplink.isSupported(SsoDeeplink.FlowVersion.REDIRECT_TO_SDK)) {
             Intent intent = LoginActivity.newIntent(
                     activity,
-                    new ArrayList<>(productFlowPriority),
+                    productFlowPriority,
```

Comment:
No longer necessary to wrap this in a new ArrayList since we are only storing defensive copy ArrayLists.